core/templates/dev/head/services/navigation-bars-backend-api.service.spec.ts

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for ReviewTestBackendApiService.
+ */
+
+// TODO(#7222): Remove the following block of unnnecessary imports once
+// the code corresponding to the spec is upgraded to Angular 8.
+import { UpgradedServices } from 'services/UpgradedServices';
+// ^^^ This block is to be removed.
+require('services/navigation-bars-backend-api.service.ts');
+
+describe('Navigation bars backend API service', function() {
+  var NavigationBarsBackendApiService = null;
+  var sampleDataResults = null;
+  var $rootScope = null;
+  var $scope = null;
```

Comment:
remove this if it's not being used anywhere (remove other similar unused variables as well)